LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2020-present, Shopify Inc.
```

Comment:
```suggestion
Copyright 2020-2022, Shopify Inc.
```

You can now add Gusto, or whatever you want as well in the license as the new copyright

Reply:
Thanks @rafaelfranca , committed this. I tried putting them next to each other (https://github.com/rubyatscale/packwerk-extensions/pull/1/commits/ccfc8c3b5e33a67fb9fe2b37fa37628163fae3c4) and then saw what bundler does (https://github.com/rubygems/bundler/blob/master/LICENSE.md) and tried to replicate that here: https://github.com/rubyatscale/packwerk-extensions/pull/1/commits/086753b695fb852aaebdce191ed82fc6d4327774

Let me know if this is what you had in mind!